scripts/run-actv-3-shex.py

```diff
@@ -0,0 +1,43 @@
+import re
```

Comment:
Can you change the file name according this naming convention? https://github.com/naming-convention/naming-convention-guides/tree/master/python